docs/root/configuration/http/http_filters/jwt_authn_filter.rst

```diff
@@ -64,6 +64,44 @@ In the :ref:`filter config <envoy_v3_api_msg_extensions.filters.http.jwt_authn.v
 
 Due to above requirement, `OpenID Connect Discovery <https://openid.net/specs/openid-connect-discovery-1_0.html>`_ is not supported since the URL to fetch JWKS is in the response of the discovery. It is not easy to setup a cluster config for a dynamic URL.
 
+
+Token Extraction from Custom HTTP Headers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the JWT needs to be extracted in other HTTP header, use ``from_headers`` to specify the header name.
```

Comment:
Can the "``from_headers``" (and other fields) be replaced with a link to the API field?
(e.g., :ref:`from_headers <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.JwtProvider. from_headers>`)

Reply:
Done